src/kafka/protocol/messages.js

```diff
@@ -202,6 +239,89 @@ const parseCommand = (messageProtocol) => {
   }
 }
 
+/**
+ * Encodes Payload to base64 encoded data URI
+ *
+ * @param {buffer\|string} input - Buffer or String
+ * @param {MimeTypes} mimeType - mime type of the input
+ *
+ * @return {string} - Returns base64 encoded data  URI string
+ */
+
+exports.encodePayload = (input, mimeType) => {
```

Comment:
Please use the same mechanism for exposing the functions by adding this in the export section on the bottom of the file